src/Catalog/CatalogTypeConverter.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace NuGet.Services.Metadata.Catalog
+{
+    public class CatalogTypeConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return objectType == typeof(IEnumerable<string>);
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            object type = null;
+            if (reader.TokenType == JsonToken.StartArray)
+            {
+                // If the type is stored in an array, use the first value and then discard the rest.
```

Comment:
>first [](start = 62, length = 5)

This comment seems wrong - doesn't this grab the *last* value?

Reply:
nice catch haha